charms/istio-pilot/src/istioctl.py

```diff
@@ -42,7 +44,8 @@ def _istioctl_flags(self):
             "--set",
             f"values.global.istioNamespace={self._namespace}",
         ]
-        istioctl_flags.extend(self._istioctl_extra_flags)
+        if self._istioctl_extra_flags:
+            istioctl_flags.extend(self._istioctl_extra_flags)
```

Comment:
Nothing against this, but imo it isn't needed since the `init` sets this private variable.  But if you want it, keep it!

Reply:
Oh, this was a change I did in the interim, I'll revert it back to how it was. Good catch!